src/CryptoHelper.php

```diff
@@ -112,7 +135,20 @@ public static function verify($data, $caInfo = null, $rootCerts = null)
 
         $outFile = self::getTempFilename();
 
-        return openssl_pkcs7_verify($data, $flags, $outFile, $rootCerts) === true;
+        $out = openssl_pkcs7_verify($data, $flags, $outFile, $rootCerts) === true;
+
+        if ($out === false && $encodeData === false) {
+            $originalCall = func_get_args();
```

Comment:
Why do you need to use `func_get_args`?

Just use the args directly `$data, $caInfo = null, $rootCerts = null`

Reply:
$data and $rootCerts is modified, or I use func_get_args or I need to put it in a variable before the update